configure.py

```diff
@@ -49,6 +49,18 @@
         dst_file="nope.media-@VERSION@.tar.gz",
         sha256="b11cad7ed46b507312410386000261797f41eaa0d7e6e108da52e5adfbc6090c",
     ),
+    egl_registry=dict(
+        version="9ab6036",
+        url="https://github.com/KhronosGroup/EGL-Registry/archive/@VERSION@.zip",
+        dst_file="egl-registry-@VERSION@.zip",
+        sha256="b981b5a250d4f10284c670ae89f4a43c9eac42adca65306e5d56b921f609c46b",
+    ),
+    opengl_registry=dict(
+        version="d0342a4",
+        url="https://github.com/KhronosGroup/OpenGL-Registry/archive/@VERSION@.zip",
+        dst_file="opengl-registry-@VERSION@.zip",
+        sha256="8e6efa8d9ec5ef1b2c735b2fc2afdfed210a3a5aa01c48e572326914e27bb221",
+    ),
```

Comment:
I believe these need to be present in the clean rules like the other packages?

Reply:
Currently there is no clean rule for renderdoc. Creating one for every package deploying binaries files means tracking the files we copy ? Do we want that ?

Reply:
I had the misconception that we were compiling something, which is not the case, so yeah we probably don't need a cleanup rule